core/es_isolate.rs

```diff
@@ -201,9 +191,10 @@ impl EsIsolate {
   /// the V8 exception. By default this type is CoreJSError, however it may be a
   /// different type if Isolate::set_js_error_create() has been used.
   fn mod_instantiate(&mut self, id: ModuleId) -> Result<(), ErrBox> {
-    let isolate = self.core_isolate.v8_isolate.as_ref().unwrap();
-    let mut locker = v8::Locker::new(isolate);
-    let mut hs = v8::HandleScope::new(locker.enter());
+    let v8_isolate = self.core_isolate.v8_isolate.as_mut().unwrap();
+    let js_error_create_fn = &*self.core_isolate.js_error_create_fn;
```

Comment:
what's happening here?

Reply:
this is "splitting the struct". The `&*` part is for borrowing out of an Arc since `js_error_create_fn` is wrapped in an Arc (which could be removed I guess).